src/components/SnackChat/SnackChat.js

```diff
@@ -78,112 +59,140 @@ class SnackChat extends Component {
-    this.ctx.drawImage(
-      video,
-      Math.abs(video.videoWidth - this.canvas.current.width) / 2 -
-        video.videoWidth,
-      0
-    );
-    this.ctx.restore();
-  };
-
   playGettingInPositionAnimation = () => {
-    const numberOfFallingSnacks = 3;
     const itemPositions = [];
+    const numberOfFallingSnacks = 3;
     for (let i = 1; i <= numberOfFallingSnacks; i++) {
       itemPositions.push({
-        x: (this.canvas.current.width * i) / (numberOfFallingSnacks + 1),
+        x: (FEED_SIZE * i) / (numberOfFallingSnacks + 1),
         y:
-          Math.random() *
-            (1 + 2 * FALLING_SNACK_SIZE) *
-            this.canvas.current.height -
-          FALLING_SNACK_SIZE * this.canvas.current.height,
+          Math.random() * (1 + 2 * FALLING_SNACK_SIZE) * FEED_SIZE -
+          FALLING_SNACK_SIZE * FEED_SIZE,
         rotation: Math.random() * 2 * Math.PI
       });
     }
 
     const drawFallingSnacks = () => {
       if (this.state.counter <= COUNTDOWN_TIME + PHOTO_ANIMATION_TIME) {
         this.setState({gettingInPosition: false});
+        requestAnimationFrame(this.update);
         return;
+      } else {
+        this.setState({itemPositions});
       }
 
-      // Video background
-      const video = this.webcam.current.webcam.current.video;
-      this.ctx.save();
-      this.drawBackground(video);
-
-      this.ctx.fillStyle = 'rgba(0.6, 0.6, 0.6, 0.6)';
-      this.ctx.fillRect(
-        0,
-        0,
-        this.canvas.current.width,
-        this.canvas.current.height
-      );
-      // draw items
-      this.ctx.save();
-      itemPositions.forEach(item => {
-        this.ctx.translate(item.x, item.y);
-        this.ctx.rotate(item.rotation);
-        this.ctx.drawImage(
-          this.filter,
-          -0.1 * this.canvas.current.width,
-          -0.1 * this.canvas.current.width,
-          FALLING_SNACK_SIZE * this.canvas.current.width,
-          FALLING_SNACK_SIZE * this.canvas.current.height
-        );
-        this.ctx.setTransform(1, 0, 0, 1, 0, 0);
-      });
-      this.ctx.restore();
-
       // update positions
       itemPositions.forEach(item => {
         item.y =
-          ((FALLING_SNACK_SIZE * this.canvas.current.height + item.y + 3) %
-            ((1 + 2 * FALLING_SNACK_SIZE) * this.canvas.current.height)) -
-          FALLING_SNACK_SIZE * this.canvas.current.height;
+          ((FALLING_SNACK_SIZE * FEED_SIZE + item.y + 3) %
+            ((1 + 2 * FALLING_SNACK_SIZE) * FEED_SIZE)) -
+          FALLING_SNACK_SIZE * FEED_SIZE;
         item.rotation = (item.rotation + 0.03) % (Math.PI * 2);
       });
-
       requestAnimationFrame(drawFallingSnacks);
     };
-
     requestAnimationFrame(drawFallingSnacks);
-    this.ctx.clearRect(
-      0,
-      0,
-      this.canvas.current.width,
-      this.canvas.current.height
+  };
+
+  clipEllipse(ctx, centerX, centerY, width, height) {
+    ctx.beginPath();
+    ctx.moveTo(centerX, centerY - height / 2);
+    ctx.bezierCurveTo(
```

Comment:
Might be useful to add a comment above both of these `bezierCurveTo` calls to remind us what they're doing, or extract some methods that `clipEllipse` calls